src/pages/partners/affiliate-ib/index.tsx

```diff
@@ -74,9 +74,12 @@ const SubtitleHeader = styled(Header)`
         width: 100%;
     }
     @media ${device.tabletL} {
-        font-size: 16px;
+        font-size: 18px;
         text-align: justify;
     }
+    @media ${device.mobileL} {
+        text-align: center;
+    }
```

Comment:
This looks like it can be avoid if you use the `subtitle-1` type instead. We don't need SubtitleHeader here, just `Header`.